src/data/data.cc

```diff
@@ -201,8 +201,10 @@ void LoadFeatureType(std::vector<std::string>const& type_names, std::vector<Feat
       types->emplace_back(FeatureType::kNumerical);
     } else if (elem == "categorical") {
       types->emplace_back(FeatureType::kCategorical);
```

Comment:
Why are we keeping this? I don't think there is a reason to keep backward compatibility here, since there isn't yet existing body of code that's using this feature.

Reply:
Removed.